client/src/main/scala/Client.scala

```diff
@@ -0,0 +1,9 @@
+import scala.scalajs.js.JSApp
+import org.scalajs.dom
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594746741/issues/source?bid=3301982&fileBranchId=3306761#l2)